ocaml/idl/datamodel_types.ml

```diff
@@ -275,7 +275,7 @@ let release_order_full = [{
      version_major = 2;
      version_minor = 15;
      branding      = "Citrix Hypervisor 8.2 Hotfix 2";
```

Comment:
Should we add a 7.1 CU2 to this list? CU1 is there, but CU2 seems to have ben forgotten

Reply:
It should probably be the other way round, i.e. CU1 should be removed. There were no public API changes either in 7.1 CU1 or 7.1 CU2 (apart from a few new API errors, which don't have version info). Indeed this page http://xapi-project.github.io/xen-api/releases/honolulu.html is empty, so I don't know why CU1 was defined.